CHANGELOG.md

```diff
@@ -81,6 +81,7 @@ Decidim::User.where(**search for old subscribed users**).update(newsletter_notif
 
 **Fixed**:
 
+- **decidim-meeting**: Fix admin update meeting with only one locale [\#4625](https://github.com/decidim/decidim/pull/4625)
```

Comment:
This should go to the unreleased section

Reply:
Thanks. No CHANGELOG entry then?